server/fishtest/util.py

```diff
@@ -592,3 +592,21 @@ def strip_run(run):
 
 def count_games(stats):
     return stats["wins"] + stats["losses"] + stats["draws"]
+
+
+def pack_flips(a):
```

Comment:
I wonder if you could use variable names more explicative, otherwise I need to reverse engineering the function with the risk to set a bad/wrong name :)